tests/testthat/test-zzz-standalone.R

```diff
@@ -0,0 +1,22 @@
+test_that("can use utils in standalone", {
+  testthat::skip_if_offline()
+  path <- withr::local_tempdir()
+  writeLines(c(
+    "Package: pkg",
+    "Version: 1.0.0"),
+    file.path(path, "DESCRIPTION"))
+  file.create(file.path(path, "NAMESPACE"))
+  dir.create(file.path(path, "R"))
+
+  evaluate_promise(
+    usethis::with_project(
+      path,
+      usethis::use_standalone("reside-ic/reside.utils", "utils-assert",
+                              ref = "prototype"))) # TODO: drop on merge
```

Comment:
```suggestion
      usethis::use_standalone("reside-ic/reside.utils", "utils-assert")))
```
as we merge, though it won't pass on that commit, be warned.

Reply:
Looks good - committing this then merging!